docs/docs/conditional-rendering.md

```diff
@@ -2,8 +2,6 @@
 id: conditional-rendering
 title: Conditional Rendering
 permalink: docs/conditional-rendering.html
-prev: handling-events.html
```

Comment:
Hmm. I'm a bit confused. These pages (e.g. Conditional Rendering) don't have to do with addons as far as I can see.